unfetter-discover-api/api/controllers/x_unfetter_assessments.js

```diff
@@ -214,27 +214,30 @@ const riskPerKillChain = controller.getByIdCb((err, result, req, res, id) => { /
                 assessment = assessment.toObject().stix;
             }
 
-            const indicators = results[0]
+            const courseOfActions = results[0]
```

Comment:
Should this be `courseOfActions` or `coursesOfAction`?

Reply:
you might have a valid point.  still this is a hotfix and would like to minimize frivolous changes outside a functional fix.  additionally one reference to the `courseOfActions` variable might propagate to the client or ui, which would require even more work and testing to change.
no más, no más!?!?